internal/compiler/llr/lower_to_item_tree.rs

```diff
@@ -545,6 +547,15 @@ fn lower_sub_component(
         crate::layout::Orientation::Vertical,
     )
     .into();
+    grid_layout_cell_for_component(component).map(|parent_grid_layout_cell| {
```

Comment:
maybe `sub_component.grid_layout_input_for_repeated = grid_layout_cell_for_component(...).map(...);` would be more readable? (I know clippy contains about Option::map returning `()`)

Reply:
Ah, I didn't know that. If only the code was clippy clean, we could enable it in the commit hook ;)